pyproject.toml

```diff
@@ -60,6 +60,12 @@ dependencies = [
     "bcrypt==4.0.1",
     "boto3==1.34.63",
     "cloudflare==2.11.7",
+    "google-auth==2.31.0",
+    "google-cloud-compute==1.19.1",
+    "google-cloud-container==2.49.0",
+    "google-cloud-iam==2.15.1",
+    "google-cloud-storage==2.18.0",
+    "grpc-google-iam-v1==0.13.1",
```

Comment:
I needed to add this library explicitly as our [conda tests where failing because of it](https://github.com/nebari-dev/nebari/actions/runs/10219550032/job/28278025426).